omero/autogen_db_version.py

```diff
@@ -16,12 +17,11 @@ def get_mmp(sqlfile):
 serverdir = sys.argv[1]
```

Comment:
@sbesson points out that using `path(sys.argv[1])` should make https://github.com/openmicroscopy/openmicroscopy/pull/6020 unnecessary.

Reply:
it does not. I have tried ``os.path.abspath`` for example and it still fails

Reply:
You need `import path` from https://github.com/openmicroscopy/openmicroscopy/blob/d66d25f33582d018551209fc33a8a076c3f6ebb8/components/tools/OmeroPy/src/omero/cli.py#L38

Reply:
That path!!